Release build fails

Trying to execute a release build on master branch currently aborts with the below errors when running `gradlew assembleRelease`.  Likely related to Proguard and issue reporting implementation, which includes a new library and dependencies there - I'm working on this.

~~~
$ gradlew assembleRelease
WARNING: Dependency org.json:json:20141113 is ignored for agencyXAmazonDebug as
it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for agencyXAmazonRelease a
s it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for agencyXGoogleDebug as
it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for agencyXGoogleRelease a
s it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for agencyYAmazonDebug as
it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for agencyYAmazonRelease a
s it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for agencyYGoogleDebug as
it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for agencyYGoogleRelease as it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for obaAmazonDebug as it may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for obaAmazonRelease as it
 may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for obaGoogleDebug as it m
ay be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
WARNING: Dependency org.json:json:20141113 is ignored for obaGoogleRelease as it
 may be conflicting with the internal version provided by Android.
         In case of problem, please repackage with jarjar to change the class packages
:onebusaway-android:copyMapsApiV2Classes UP-TO-DATE
:onebusaway-android:preBuild UP-TO-DATE
:onebusaway-android:preAgencyXAmazonReleaseBuild UP-TO-DATE
:onebusaway-android:checkAgencyXAmazonReleaseManifest
:onebusaway-android:preAgencyXAmazonDebugBuild UP-TO-DATE
:onebusaway-android:preAgencyYAmazonDebugBuild UP-TO-DATE
:onebusaway-android:preAgencyYAmazonReleaseBuild UP-TO-DATE
:onebusaway-android:preObaAmazonDebugBuild UP-TO-DATE
:onebusaway-android:preObaAmazonReleaseBuild UP-TO-DATE
:onebusaway-android:prepareComAmazonAndroidAmazonMapsApi20Library
:onebusaway-android:preAgencyXGoogleDebugBuild UP-TO-DATE
:onebusaway-android:preAgencyXGoogleReleaseBuild UP-TO-DATE
:onebusaway-android:preAgencyYGoogleDebugBuild UP-TO-DATE
:onebusaway-android:preAgencyYGoogleReleaseBuild UP-TO-DATE
:onebusaway-android:preObaGoogleDebugBuild UP-TO-DATE
:onebusaway-android:preObaGoogleReleaseBuild UP-TO-DATE
:onebusaway-android:prepareComAndroidSupportAppcompatV72221Library UP-TO-DATE
:onebusaway-android:prepareComAndroidSupportCardviewV72221Library UP-TO-DATE
:onebusaway-android:prepareComAndroidSupportDesign2221Library UP-TO-DATE
:onebusaway-android:prepareComAndroidSupportSupportV42311Library UP-TO-DATE
:onebusaway-android:prepareComGithubAmlcurranShowcaseviewLibrary543Library UP-TO-DATE
:onebusaway-android:prepareComGoogleAndroidGmsPlayServicesAnalytics830Library UP-TO-DATE
:onebusaway-android:prepareComGoogleAndroidGmsPlayServicesBase830Library UP-TO-DATE
:onebusaway-android:prepareComGoogleAndroidGmsPlayServicesBasement830Library UP-TO-DATE
:onebusaway-android:prepareComGoogleAndroidGmsPlayServicesLocation830Library UP-TO-DATE
:onebusaway-android:prepareComGoogleAndroidGmsPlayServicesMaps830Library UP-TO-DATE
:onebusaway-android:prepareComSothreeSlidinguppanelLibrary311Library UP-TO-DATE
:onebusaway-android:prepareAgencyXAmazonReleaseDependencies
:onebusaway-android:compileAgencyXAmazonReleaseAidl
:onebusaway-android:compileAgencyXAmazonReleaseRenderscript
:onebusaway-android:generateAgencyXAmazonReleaseBuildConfig
:onebusaway-android:generateAgencyXAmazonReleaseAssets UP-TO-DATE
:onebusaway-android:mergeAgencyXAmazonReleaseAssets
:onebusaway-android:generateAgencyXAmazonReleaseResValues UP-TO-DATE
:onebusaway-android:generateAgencyXAmazonReleaseResources
:onebusaway-android:mergeAgencyXAmazonReleaseResources
AAPT err(56705202): F:\Android_Studio\onebusaway-android\onebusaway-android\src\main\res\drawable-mdpi\ic_light_bulb.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(1470143038): F:\Android_Studio\onebusaway-android\onebusaway-android\src\main\res\drawable-hdpi\ic_light_bulb.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(1215593490): F:\Android_Studio\onebusaway-android\onebusaway-android\src\main\res\drawable-hdpi\ic_customer_service_phone.png: libpng warning: iCCP: No
t recognizing known sRGB profile that has been edited
AAPT err(1610433570): F:\Android_Studio\onebusaway-android\onebusaway-android\src\main\res\drawable-xxhdpi\ic_light_bulb.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
AAPT err(2047991957): F:\Android_Studio\onebusaway-android\onebusaway-android\src\main\res\drawable-xhdpi\ic_light_bulb.png: libpng warning: iCCP: Not recognizing known sRGB profile that has been edited
:onebusaway-android:processAgencyXAmazonReleaseManifest
:onebusaway-android:processAgencyXAmazonReleaseResources
Unknown source file : warning: string 'report_problem_cancel' has no default translation.
Unknown source file : warning: string 'report_problem_code' has no default translation.
Unknown source file : warning: string 'report_problem_comment' has no default translation.
Unknown source file : warning: string 'report_problem_email' has no default translation.
Unknown source file : warning: string 'report_problem_email_hint' has no default
 translation.
Unknown source file : warning: string 'report_problem_vehiclenumber_bus' has no
default translation.
Unknown source file : warning: string 'stop_info_option_reportproblem' has no default translation.

:onebusaway-android:generateAgencyXAmazonReleaseSources
:onebusaway-android:processAgencyXAmazonReleaseJavaRes UP-TO-DATE
:onebusaway-android:compileAgencyXAmazonReleaseJavaWithJavac
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
:onebusaway-android:compileAgencyXAmazonReleaseNdk UP-TO-DATE
:onebusaway-android:compileAgencyXAmazonReleaseSources
> Building 25% > :onebusaway-android:askForPasswords
Keystore password:
Key password:
:onebusaway-android:askForPasswords
:onebusaway-android:proguardAgencyXAmazonRelease
Note: there were 12 duplicate class definitions.
      (http://proguard.sourceforge.net/manual/troubleshooting.html#duplicateclass)
Warning: library class android.test.AndroidTestCase extends or implements program class junit.framework.TestCase
Warning: library class android.test.AndroidTestRunner extends or implements prog
ram class junit.runner.BaseTestRunner
Warning: library class android.test.InstrumentationTestCase extends or implement
s program class junit.framework.TestCase
Warning: library class android.test.InstrumentationTestSuite extends or implements program class junit.framework.TestSuite
Warning: library class android.test.suitebuilder.TestSuiteBuilder$FailedToCreate
Tests extends or implements program class junit.framework.TestCase
Warning: org.junit.internal.runners.statements.FailOnTimeout: can't find referenced class java.lang.management.ManagementFactory
Warning: org.junit.internal.runners.statements.FailOnTimeout: can't find referenced class java.lang.management.ThreadMXBean
Warning: org.junit.internal.runners.statements.FailOnTimeout: can't find referenced class java.lang.management.ThreadMXBean
Warning: org.junit.internal.runners.statements.FailOnTimeout: can't find referenced class java.lang.management.ManagementFactory
Warning: org.junit.internal.runners.statements.FailOnTimeout: can't find referenced class java.lang.management.ThreadMXBean
Warning: org.junit.internal.runners.statements.FailOnTimeout: can't find referenced class java.lang.management.ThreadMXBean
Warning: org.junit.rules.DisableOnDebug: can't find referenced class java.lang.management.ManagementFactory
Warning: org.junit.rules.DisableOnDebug: can't find referenced class java.lang.management.RuntimeMXBean
Warning: org.junit.rules.DisableOnDebug: can't find referenced class java.lang.management.ManagementFactory
Warning: org.junit.rules.DisableOnDebug: can't find referenced class java.lang.management.RuntimeMXBean
Warning: there were 10 unresolved references to classes or interfaces.
         You may need to add missing library jars or update their versions.
         If your code works fine without the missing classes, you can suppress
         the warnings with '-dontwarn' options.
         (http://proguard.sourceforge.net/manual/troubleshooting.html#unresolved
class)
Warning: there were 5 instances of library classes depending on program classes.

         You must avoid such dependencies, since the program classes will
         be processed, while the library classes will remain unchanged.
         (http://proguard.sourceforge.net/manual/troubleshooting.html#dependency
)
Exception while processing task
java.io.IOException: Please correct the above warnings first.
        at proguard.Initializer.execute(Initializer.java:473)
        at proguard.ProGuard.initialize(ProGuard.java:233)
        at proguard.ProGuard.execute(ProGuard.java:98)
        at proguard.gradle.ProGuardTask.proguard(ProGuardTask.java:1074)
        at com.android.build.gradle.tasks.AndroidProGuardTask.doMinification(And
roidProGuardTask.java:139)
        at com.android.build.gradle.tasks.AndroidProGuardTask$1.run(AndroidProGu
ardTask.java:115)
        at com.android.builder.tasks.Job.runTask(Job.java:48)
        at com.android.build.gradle.tasks.SimpleWorkQueue$EmptyThreadContext.run
Task(SimpleWorkQueue.java:41)
        at com.android.builder.tasks.WorkQueue.run(WorkQueue.java:227)
        at java.lang.Thread.run(Thread.java:745)
:onebusaway-android:dexAgencyXAmazonRelease
:onebusaway-android:shrinkAgencyXAmazonReleaseResources
Removed unused resources: Binary resource data reduced from 2185KB to 973KB: Rem
oved 55%
Note: If necessary, you can disable resource shrinking by adding
android {
    buildTypes {
        release {
            shrinkResources false
        }
    }
}
:onebusaway-android:validateReleaseSigning
:onebusaway-android:packageAgencyXAmazonRelease FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':onebusaway-android:packageAgencyXAmazonRelease'.
> Unable to compute hash of F:\Android_Studio\onebusaway-android\onebusaway-andr
oid\build\intermediates\classes-proguard\agencyXamazon\release\classes.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug
option to get more log output.

BUILD FAILED

Total time: 1 mins 4.5 secs
~~~

--- a/onebusaway-android/build.gradle
+++ b/onebusaway-android/build.gradle
@@ -295,7 +295,7 @@ dependencies {
     compile 'com.android.support:design:22.2.1'
     compile 'commons-io:commons-io:2.4'
     // Open311 client library
-    compile 'edu.usf.cutr:open311client:1.0.7'
+    compile 'edu.usf.cutr:open311client:1.0.8'
     // JSON data binding for OBA REST API responses
     compile 'com.fasterxml.jackson.core:jackson-core:2.6.2'
     compile 'com.fasterxml.jackson.core:jackson-annotations:2.6.2'

